docs/pages/reference/mesheryctl/commands/mesheryctl_command_template.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Command Doc Title
+permalink: reference/mesheryctl/commands/template
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+<!-- 
+ This could be used as a template for commands documentation.
+ Just fill your information in the sections indicated below.
+ -->
+
+
+
+<!-- name of command -->
+# mesheryctl
+<pre class="codeblock-pre">
+ <div class="codeblock"><div class="clipboardjs">
+<!-- General example of command -->
+ mesheryctl &#60;command&#62; [flags]
+ </div></div>
+ </pre>
+
+<!-- General Synopse of command -->
+<!-- General Description of the command -->
+Describe the command goes here.
+<hr/>
+<br/>
+
+<!-- General usage of command -->
+## Examples
+
+ <pre class="codeblock-pre">
+ <div class="codeblock"><div class="clipboardjs">
+ <!-- Command Examples goes here -->
+  mesheryctl 
+ </div></div>
+ </pre>
+
+**Outputs** 
+ <pre class="codeblock-pre">
+ <div class="codeblock"><div class="clipboardjs">
+<!-- Output examples goes here -->
+
+ </div></div>
+ </pre>
+<!-- extra doc could be here -->
+When you execute `mesheryctl` without any flag, the output would be the usage and general help info about the command.
+
+
+
+## Options 
+- **_(none)_**
+When you execute `mesheryctl` without any flag, the output would be the usage and general help info about the command.
+<br/>
+- **_(a)_**
+Description for 'a' lorem ipsum.
+<br/>
+- **_(b)_**
+Description for 'b' lorem ipsum
+<br/>
+- **_(c)_**
+Description for 'c' lorem ipsum
+<br/>
+<br/>

```

Comment:
Options should only have a code block as seen [here](https://cli.github.com/manual/gh_auth_logout)